src/pages/DataSharingLanguageTool.js

```diff
@@ -0,0 +1,232 @@
+import {RadioButton} from "../components/RadioButton";
+import AsyncSelect from "react-select/async/dist/react-select.esm";
+import {isNil} from "lodash/fp";
+import {Notifications, searchOntologies} from "../libs/utils";
+import {DataUseTranslation} from "../libs/dataUseTranslation";
+import {useState} from "react";
+
+const buttonStyle = {marginBottom: '2rem', color: '#777'};
+
+export default function DataSharingLanguageTool(props) {
+  const [general, setGeneral] = useState(false);
+  const [hmb, setHmb] = useState(false);
+  const [diseases, setDiseases] = useState(false);
+  const [ontologies, setOntologies] = useState();
+  const [other, setOther] = useState(false);
+  const [otherText, setOtherText] = useState();
+  const [nmds, setNmds] = useState(false);
+  const [gso, setGso] = useState(false);
+  const [pub, setPub] = useState(false);
+  const [col, setCol] = useState(false);
+  const [irb, setIrb] = useState(false);
+  const [gs, setGs] = useState(false);
+  const [mor, setMor] = useState(false);
+  const [npu, setNpu] = useState(false);
+  const [sdsl, setSdsl] = useState("");
+
+  const isTypeOfResearchValid = () => {
+    return (general || hmb || (diseases && !isNil(ontologies)) || other );
+  };
+
+  const generate = () => {
+    isTypeOfResearchValid() ?
+      generateHelper()
+      : Notifications.showError({text: "Please complete question 1"});
+  };
+
+  const generateHelper = () => {
+    const darInfo = {
+      general: general, diseases: diseases, ontologies: ontologies, other: other, otherText: otherText,
+      hmb: hmb, methods: nmds, forProfit: !npu
+    };
+    const summaries = [];
+    const dataUse = (DataUseTranslation.translateDarInfo(darInfo));
+    dataUse.primary.forEach((summary) => {
+      summaries.push(summary.description);
+    });
+    dataUse.secondary.forEach((summary) => summaries.push(summary.description));
+    setSdsl(summaries.join(" "));
+  };
+
+  return (
+    div({style: {...Styles.PAGE, color: '#1f3b50' }}, [
+      div({style: {...Styles.TITLE, marginTop: '3.5rem'}}, [
+        "Standardized Data Sharing Language Tool"
+      ]),
+      div({style: {...Styles.SMALL, marginTop: '1rem'}}, [
+        "This tool is made publicly available by the DUOS team for anyone" +
+        " interested in leveraging standardized data sharing language in their" +
+        " consent forms. The tool leverages the Global Alliance for Genomics " +
+        "and Health’s (GA4GH) companion standards of the Data Use Ontology (DUO) " +
+        "and Machine Readable Consent Guidance (MRCG). The DUO is a structured " +
+        "vocabulary describing permitted data uses and the MRCG is a suggested " +
+        "representation of those uses in consent form language. This tool enables" +
+        " users to easily define what types of data use they would like permitted" +
+        " in their consent forms and then suggests corresponding text for the " +
+        "consent form below, based on the MRCG."
```

Comment:
Might be outside the scope of the story, but it would be nice to have links here to the standards that we're describing. Let's follow up with @solideoglori about adding some link content here.

Reply:
I believe that info in in the MRCG link!